docs/docs/50-user-guide/70-annotations.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_label: Annotations
+description: List of Kargo-specific annotations and labels
+---
+
+# Annotations and Labels used by Kargo
+
+This page documents Kargo-specific annotations, labels, and finalizers. Some of these fields can be set by users, while others are system-managed and used internally by the control plane.
+
+## Annotations
+
+| Annotation key | Target resource(s) | Possible values | Description |
+|----------------|--------------------|------------------|-------------|
+| `kargo.akuity.io/refresh` | Any Kargo resource | Arbitrary string (acts as a token) | Triggers reconciliation when the value changes. Useful for forcing a resync. |
+| `kargo.akuity.io/reverify` | `Stage` | Verification ID or JSON object (VerificationRequest) | Triggers re-verification of the currently assigned Freight. |
+| `kargo.akuity.io/abort` | `Stage` | Verification ID | Aborts an in-progress Freight verification. |
+| `kargo.akuity.io/description` | Any | Free-form string | Human-readable description displayed in the UI. |
+| `kargo.akuity.io/color` | `Stage` | Hex color code (e.g. `#ff8800`) | Optional cosmetic color used in UI. |
+| `kargo.akuity.io/create-actor` | All | Actor identity (username/service) | Injected by the control plane to indicate who created the resource. |
+| `kargo.akuity.io/authorized-stage` | `Freight`, `Promotion`, etc. | `<project>:<stage>` | Indicates which Stage is authorized to manage the resource. |
+| `kargo.akuity.io/promotion` | `PromotionTask`, internal resources | Promotion ID or name | Links the resources to a promotion process. |
+| `kargo.akuity.io/argocd-context` | `Stage` | JSON or internal identifier | Records Argo CD application context from the last promotion. |
+
+:::warning
+Avoid setting system-managed annotations manually unless you're explicitly debugging internal behavior.
+:::
+
+### RBAC Annotations
+
+| Annotation key | Target resource(s) | Possible values | Description |
+|----------------|--------------------|------------------|-------------|
+| `rbac.kargo.akuity.io/managed` | RBAC resources | `"true"` | Marks a resource as managed by Kargo's RBAC system. |
+| `rbac.kargo.akuity.io/claim.<name>` | RBAC resources | Claim name as suffix | Maps an OIDC claim to a policy. |
```

Comment:
This annotation is only used on ServiceAccounts, which are not RBAC resources.

I would also avoid the term "policy," since that term doesn't accurately describe a ServiceAccount.

This annotation is already well-documented [here](https://deploy-preview-3998.docs.kargo.io/operator-guide/security/access-controls#user-to-serviceaccount-mappings) and [here](https://docs.kargo.io/user-guide/security/access-controls/#user-to-serviceaccount-mappings). Please read up to understand how this works and try to improve the accuracy of this row. You can also link from here to those more thorough explanations of this annotation's effects.